modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -125,10 +125,16 @@ public class CatalogUtils {
     private static final Map<ColumnType, Set<ColumnType>> ALTER_COLUMN_TYPE_TRANSITIONS = new EnumMap<>(ColumnType.class);
 
     static {
-        ALTER_COLUMN_TYPE_TRANSITIONS.put(ColumnType.INT8, EnumSet.of(ColumnType.INT16, ColumnType.INT32, ColumnType.INT64));
```

Comment:
Maybe we can initialise this map of supported conversions for all types so that for every type `T` `isSupportedColumnTypeChange(T, T)` always returns`true` ?

Reply:
fix a bit here: validateColumnChange